agent/grpc-external/services/dataplane/get_envoy_bootstrap_params_test.go

```diff
@@ -280,3 +364,185 @@ func TestGetEnvoyBootstrapParams_PermissionDenied(t *testing.T) {
 	require.Equal(t, codes.PermissionDenied.String(), status.Code(err).String())
 	require.Nil(t, resp)
 }
+
+// Copied implementation from agent/consul/merge_service_config.go to avoid import cycle.
+func testMergeNodeServiceWithCentralConfig(
```

Comment:
I copied the function over for the unit tests because I didn't know a better way to avoid the import cycle.